ee/localserver/server.go

```diff
@@ -83,37 +83,18 @@ func New(logger log.Logger, db *bbolt.DB, kolideServer string) (*localServer, er
 	}
 	ls.myKey = privateKey
 
-	// Setup the krypto boxer middleware. This will be used for the v1 protocol
-	kbm, err := NewKryptoBoxerMiddleware(ls.logger, ls.myKey, ls.serverKey)
-	if err != nil {
-		return nil, fmt.Errorf("creating krypto boxer middlware: %w", err)
-	}
-
-	rsaAuthedMux := http.NewServeMux()
-	rsaAuthedMux.HandleFunc("/", http.NotFound)
-	rsaAuthedMux.HandleFunc("/ping", pongHandler)
-	rsaAuthedMux.Handle("/id", kbm.Wrap(ls.requestIdHandler()))
-	rsaAuthedMux.Handle("/id.png", kbm.WrapPng(ls.requestIdHandler()))
-
-	// Setup the v2 protocol wraps
 	ecKryptoMiddleware := newKryptoEcMiddleware(ls.logger, ls.myLocalDbSigner, ls.myLocalHardwareSigner, *ls.serverEcKey)
-
 	ecAuthedMux := http.NewServeMux()
+	ecAuthedMux.HandleFunc("/", http.NotFound)
+	ecAuthedMux.HandleFunc("/ping", pongHandler)
```

Comment:
You can drop these. We don't use them, and they didn't work with the EC middleware.